### ISSUE TITLE
Grant Task 3b: Quantitative model comparison

### ISSUE THREAD

**Issue by jonc125** [ModellingWebLab/project_issues#7]
Where multiple competing models can be fitted to the same data set in Task 2a, we will investigate the use of these fits with relevant information criteria to provide numerical scores (e.g. AIC, BIC, Bayes factors) for how well each model can explain the data (relative to the model’s complexity), and hence support users further in choosing the most appropriate model for their scientific question.

**Comment by mirams** [comment]
Yeah, I think that the algorithms that do this nicely might also need to fit the parameters of all the possible model structures at the same time (reversible [model structure]-jump MCMC for instance). So model selection might want to get tied up in the actual 'Fitting'/'Development' task.
